docs/tutorial/offscreen-rendering.md

```diff
@@ -57,3 +57,16 @@ app.once('ready', () => {
 ```
 
 [disablehardwareacceleration]: ../api/app.md#appdisablehardwareacceleration
+
+## User Input
+
+When a window is rendered offscreen, Electron has no way to route user input to
```

Comment:
> Electron has no way to route user input

This line contradicts the explanation given below on how to route user input, I think what this is trying to say is something like "Off screen windows do not receive user input by default, if your window requires user input you will need to route it there manually."